app/models/generic_event.rb

```diff
@@ -1,7 +1,29 @@
 class GenericEvent < ApplicationRecord
   CREATED_BY_OPTIONS = %w[serco geoamey unknown].freeze
```

Comment:
is it necessary to hard-code the supplier's names? could we use supplier_id instead?

Reply:
Was just what we agreed over calls. I think this is an iteration on the final thing, tbh. Happy to talk about it

Reply:
We could use the supplier_id, yeah. I'd be happy to do that and if the hub want the nomis agency id, etc, in the feed that can have it.

Reply:
Speaking with Martyn, we're going to change this to be more consistent with our approach elsewhere. I'll flag up in the reporting doc that we're changing this to be consistent and talk to @jcavell

Reply:
In a separate PR, too

Reply:
👍